elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/DefaultQueryValidator.java

```diff
@@ -65,10 +66,21 @@ public void validateHavingClause(Query query) {
             validatePredicate(query, predicate);
 
             extractFilterProjections(query, havingClause).stream().forEach(projection -> {
-                if (query.getColumnProjection(projection.getAlias(), projection.getArguments()) == null) {
+
+                Predicate<ColumnProjection> filterByNameAndArgs =
+                        (column) -> (column.getAlias().equals(projection.getAlias())
```

Comment:
should there be an additional ( ) around the Left side and Right side expression of || to make sure that condition is true before &&

Reply:
Yes - this is a bug (and why the build failed).